cargo-concordium/src/build.rs

```diff
@@ -422,7 +423,18 @@ pub fn build_and_run_wasm_test(extra_args: &[String]) -> anyhow::Result<bool> {
 
     eprintln!("\n{}", Color::Green.bold().paint("Running tests ..."));
 
-    let results = utils::run_module_tests(&wasm)?;
+    let seed_u64 = match seed {
+        Some(s) => s,
+        None => {
+            // this is a bit awkward, but `SmallRng::from_entropy()` constructs
+            // the whole RNG, so the seed is not visible from the outside
```

Comment:
I am not sure that I understand this comment, nor that we will understand it in a few months. Please add a bit more detail 😊

Reply:
Done